pkgs/tools/misc/online-judge-tools/default.nix

```diff
@@ -0,0 +1,42 @@
+{ git, lib, python3Packages, ... }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "online-judge-tools";
+  version = "11.5.1";
+  format = "wheel";
+
+  src = python3Packages.fetchPypi {
+    inherit version format;
+    pname = "online_judge_tools";
+    dist = "py3";
+    python = "py3";
+    sha256 = "18avv6icax7mxnk8mbhpg1if5myadplr97pwfak5yw9hnh9z97r0";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    online-judge-api-client
+    colorama
+    requests
+  ];
+
+  # Wheels have no tests
+  doCheck = false;
+
+  preFixup = ''
+    makeWrapperArgs=(
+      --prefix PATH : ${lib.makeBinPath [ git ]}
```

Comment:
The most proper fix would be to patch the source code, so that it works even when being used as a module